libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -185,7 +217,68 @@ export const TeaserAuthors = styled('div')`
 
 export const TeaserContent = styled(TeaserInnerWrapper)``
 
+export const ImagePlaceholder = styled('div')`
+  width: 100%;
+  object-fit: cover;
+  grid-column: 1/13;
+  aspect-ratio: 1/1;
+`
+
+export const TeaserContentStyled = styled(TeaserContent)`
+  grid-column: 1/13;
+`
+
+export const TeaserPreTitleNoContent = styled(TeaserPreTitle)<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  background-color: ${({theme, isHovered}) =>
+    isHovered ? theme.palette.primary.main : theme.palette.common.black};
+  height: 3px;
+  width: 100%;
+  margin-bottom: ${({theme}) => theme.spacing(1.5)};
+`
+
+export const TeaserPreTitleStyled = styled(TeaserPreTitle)<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  background-color: ${({theme, isHovered}) =>
+    isHovered ? theme.palette.primary.main : theme.palette.secondary.main};
+  height: 3px;
+  width: 100%;
+  margin-bottom: ${({theme}) => theme.spacing(1.5)};
+`
+
+export const PreTitleStyled = styled('span')<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  padding: ${({theme}) => `${theme.spacing(0.5)} ${theme.spacing(2)}`};
+  background-color: ${({theme, isHovered}) =>
+    isHovered ? theme.palette.primary.main : theme.palette.secondary.main};
+  display: inline-block;
+  font-size: 14px;
+  font-weight: 300;
+  transform: ${({theme}) => `translateY(-${theme.spacing(3)})`};
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    font-size: 18px;
+    transform: ${({theme}) => `translateY(-${theme.spacing(3.5)})`};
+  }
+`
+
+export const TeaserTitlesStyled = styled(TeaserTitle)`
+  margin: 0;
+`
+
+export const AuthorsAndDate = styled('p')`
+  margin: 0;
+  font-size: 12px;
+`
+
+export const TeaserLeadStyled = styled('div')`
```

Comment:
a paragraph so probably should be one. You should be able to use the Paragraph from website builder.